plugins/googlesignin/views/settings.twig

```diff
@@ -0,0 +1,11 @@
+<h1>{{ Title }} </h1>
+<div class="padded alert alert-warning">
+    {{ Instructions|raw }}
+</div>
+ {{ form.open()|raw }}
+ {{ form.errors() }}
+ {{ form.simple(formData)|raw }}
+<div class="modal-footer js-modal-fixed">
+{{ form.button('Save')|raw }}
+</div>
+ {{ form.close()|raw }}
```

Comment:
Too much usage of `raw` format. That makes twig less useful. 
One of the purpose of using templating library (such as twig) is to avoid XSS vulnerability, when templating engine is capable to auto escape special html characters and reduce vulnerability level.

I think that is ok to go for this PR. But we need to think about converting or extending base classes.
For example in this particular case we need `form->errors()` to return plain array with messages without any html.
And then decorate those messages with html twig template.